base/src/main/java/com/smartdevicelink/managers/file/BaseFileManager.java

```diff
@@ -383,19 +383,27 @@ private boolean shouldReUploadFile(String fileName, FileType fileType) {
 
 	/**
 	 * Attempts to upload a list of SdlFiles to core
-	 * @param files list of SdlFiles with file name and one of A) fileData, B) Uri, or C) resourceID set
+	 * @param files    list of SdlFiles with file name and one of A) fileData, B) Uri, or C) resourceID set
```

Comment:
```suggestion
	 * @param files list of SdlFiles with file name and one of A) fileData, B) Uri, or C) resourceID set
```
There are some extra spaces here

Reply:
It was the code reformat, Fixed it.